src/_includes/curated_publications.html

```diff
@@ -17,40 +17,42 @@
     {% include hero_teaser.html class="curated-publications__hero" content_type=content_type cta=cta cta-href=cta-href description=description image_position="right" image_src=image_src title=title %}
   {% endfor %}
 
-  <div class="curated-publications__cards cards-layout">
-    <div class="cards-layout__inner">
-      {% assign posts = site.posts | where: 'pinned_locations', include.pinned_location %}
-      {% for post in posts limit: 5 %}
-        {% if site.lang == "es" %}
-          {% assign article_title = post.title-es %}
-          {% assign article_description = post.description-es %}
-        {% else %}
-          {% assign article_title = post.title %}
-          {% assign article_description = post.description %}
-        {% endif %}
-        <article class="cards-layout__card card">
-          <a class="card__image-link" href="{{ post.url | prepend: site.baseurl }}">
-            <span class="card__image" style="background-image: url({{post.image.src | prepend: site.baseurl}})"></span>
-          </a>
-          <div class="card__main">
-            <h3 class="card__title">{{ article_title }}</h3>
-            <div class="card__description-fade-out">
-              <p class="card__description">{{ article_description }}</p>
+  {% if posts.size > 1 %}
```

Comment:
On a small and medium device we rely on the posts output in this block in order to show the first post. This needs to be fixed